Source/santad/SNTEventLog.m

```diff
@@ -226,15 +226,29 @@ - (void)logDiskAppeared:(NSDictionary *)diskProperties {
                         diskProperties[@"DADeviceModel"] ?: @""];
   model = [model stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]];
 
-  LOGI(@"action=DISKAPPEAR|mount=%@|volume=%@|bsdname=%@|fs=%@|model=%@|serial=%@|bus=%@|dmgpath=%@",
+  double appearance = [diskProperties[@"DAAppearanceTime"] doubleValue];
+  double now = [NSDate date].timeIntervalSinceReferenceDate;
+
+  NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+  dateFormatter.dateFormat = @"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
+  dateFormatter.timeZone = [NSTimeZone timeZoneWithName:@"UTC"];
+  NSString *appearanceDateString =
+      [dateFormatter stringFromDate:[NSDate dateWithTimeIntervalSinceReferenceDate:appearance]];
+
+  NSString *log =
+      @"action=DISKAPPEAR|%@"
+      @"mount=%@|volume=%@|bsdname=%@|fs=%@|model=%@|serial=%@|bus=%@|dmgpath=%@|appearance=%@";
+  LOGI(log,
+       (appearance && (int)appearance != (int)now) ? @"reason=NOTRUNNING|" : @"",
```

Comment:
The check that appearance is nonzero is unnecessary as now will never be zero. Also the casts to int shouldn't be needed.